### ISSUE TITLE
Iron Router infinity reloads with polymer.

### ISSUE THREAD

**Issue by laosb** [iron-meteor/iron-router#1052]
It seems that importing polymer elements by `<link rel="import" href="bower_components/xxx/xxx.html">` will let the iron router infinity reload some times. For example, `https://piet.cn//` (Must use double `/` to show this problem.)

**Comment by SylarRuby** [comment]
@ThaumRystra Thank you, thank you, thank you! I had the same issue here: http://goo.gl/fL6JIK and your findings really helped. Thank you so flipping much!

**Comment by ThaumRystra** [comment]
@SylarRuby awesome, so does this solve #1323?

**Comment by SylarRuby** [comment]
Yes it has. Im very happy.

**Comment by laosb** [comment]
It's surprise to see more discuss here. I've already figured it out but forgotten to tell you what the problem is. Sorry for my fault. 

Here's more detail about this issue.

If you try to include some `.html`s which didn't exist, you will get the infinite loop. Solution is simply delete the `<link>` tag which imports a missing html. But it shouldn't be a infinite loop, I guess. 

I'm no longer using `iron:router` for routing now, since `meteorhacks:flow-layout` might be a simple and clear solution for my app. Still, `iron:router` is a wonderful router solution. Also, I learned a lot in dealing with these bugs, and now I am able to play well with Meteor. Thank you @cmather for your hard work in making a easy-to-learn and powerful router solution!
